service/handlers.go

```diff
@@ -84,6 +89,23 @@ func VersionHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// HealthHander is the API method to return if the app is up and db.Ping() doesn't return an error
+func HealthHander(w http.ResponseWriter, r *http.Request) {
```

Comment:
Minor comment: Hander should be Handler
However, we are in the process of a refactor and removing the team Handler altogether from these functions.